footprint/risc-v/run_suite.py

```
@@ -0,0 +1,233 @@
+    ("rsa1024", "1024"),
+    ("rsa1536", "1536"),
+    ("rsa2048", "2048"),
+    ("rsa3072", "3072"),
+    ("rsa4096", "4096"),
+]
+TIMEOUT_RUN = 300  # seconds per QEMU run (4096-bit can take a while)
+TIMEOUT_BUILD = 600  # seconds for cargo build
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+TARGET_DIR = os.path.join(tempfile.gettempdir(), "rsa_footprint_riscv")
+SYSROOT = subprocess.run(
+    ["rustc", "--print", "sysroot"], capture_output=True, text=True, check=True
```

Ruff (0.15.13)

[error] Starting a process with a partial executable path

(S607)